src/movieData.js

```diff
@@ -0,0 +1,326 @@
+const movieData = {
```

Comment:
We can probably delete this movieData file since we're fetching our data

Reply:
yup!